Dockerfile

```diff
@@ -3,12 +3,8 @@ FROM quay.io/centos/centos:stream9
 # Help people find the actual baremetal command
 COPY scripts/openstack /usr/bin/openstack
 
-RUN dnf install -y python3 python3-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/plugins/module_utils/tripleo_repos/main.py | python3 - -b master current-tripleo && \
-    dnf update -y --setopt=install_weak_deps=False && \
-    dnf install -y --setopt=install_weak_deps=False python3-ironicclient python3-ironic-inspector-client && \
-    dnf clean all && \
-    rm -rf /var/cache/{yum,dnf}/* && \
+RUN dnf install -y python3 python3-requests python3-pip && \
+    pip install python-ironicclient[cli] python-ironic-inspector-client --no-cache-dir   && \
```

Comment:
We don't really need the `cli` extras: it provides the generic `openstack` CLI while we should be fine with the ironic-specific `baremetal` one.

Reply:
Got it. I wasn't entirely sure what elements were needed in the image, but thanks for the clarification. I'll remove the cli extras.